practice-one/pages/index.html

```diff
@@ -533,7 +533,7 @@ <h2 class="card-title">Memphis Design: the defining look of the 1980s</h2>
                 Exercitationem, illo.</p>
             </div>
             <div class="card-footer">
-              <button class="btn-secondary btn-primary">Read More</button>
+              <button class="btn btn-primary">Read More</button>
```

Comment:
This button is the same as the button **Get Start Now**, but you are using 2 naming conversions for them. Why?

Reply:
I created a frame button so I add class btn-primary just want it to have a different color